spec/index.bs

```diff
@@ -4782,18 +4802,27 @@ command encoder can no longer be used.
                 descriptor:
             </pre>
 
-            **Returns:** {{undefined}}
-
-            Issue the following steps on the [=Device timeline=] of |this|:
-            <div class=device-timeline>
-                1. If any of the following conditions are unsatisfied, generate a validation
-                    error and stop.
-                    <div class=validusage>
-                        - |this| is [=valid=].
-                        - |this|.{{GPUCommandEncoder/[[debug_group_stack]]}}.length is 0.
+            **Returns:** {{GPUCommandBuffer}}
+
+            Issue the following steps on the [=Content timeline=] of |this|:
+            <div class=content-timeline>
+                1. Let |commandBuffer| be a new {{GPUCommandBuffer}}.
+                1. Issue the following steps on the [=Device timeline=] of |this|:
```

Comment:
should this be separated from the `div class=content-timeline`? i.e. can we have the two divs side by side instead of one within another?